helm/ph-ee-engine/connector-channel/values.yaml

```diff
@@ -26,7 +26,15 @@ redis:
   idempotency:
     enabled: true
     keyFormat: "clientCorrelationId_tenant_api"
-    apiList: "/channel/transfer,/channel/collection,/channel/gsma/transaction,/channel/transactionRequest" 
+    apiList: "/channel/transfer,/channel/collection,/channel/gsma/transaction,/channel/transactionRequest"
+server:
+  ssl:
+    key_alias: "tomcat-https"
+    key_store: "classpath:keystore.jks"
```

Comment:
these should be taken from the application.yml if not mentioned. so don't add default values here, also make sure that these can be modified from here if needed

Reply:
make sure to test this helm chart with latest channel-coonnector and not the latest released version(v1.5.1). Since this config change is coupled with code change.

Reply:
@dhruvsonagara , yes it is added here. but these values will override the values that are coming from the Application yml. in most cases ( at least in testing /demo) the application defaults need not be overridden.

Reply:
@fynmanoj Should I remove this then?

Reply:
yes